Formula/idris2.rb

```diff
@@ -1,8 +1,8 @@
 class Idris2 < Formula
   desc "Pure functional programming language with dependent types"
   homepage "https://www.idris-lang.org/"
-  url "https://github.com/idris-lang/Idris2/archive/v0.3.0.tar.gz"
-  sha256 "2b1a921c3b46eec629936579a93319b2adb1c66a61302cd5f0b53017a07b1b74"
+  url "https://github.com/idris-lang/Idris2/archive/v0.4.0.tar.gz"
+  sha256 "2517f8462a4b15c2b71eb55655a4ec53281d7fd1e2796778dec5fcc0681094b7"
```

Comment:
Did you get this checksum from piping `curl` into `sha256sum`, or similar?
```
❯ curl -s https://github.com/idris-lang/Idris2/archive/v0.4.0.tar.gz | sha256sum
2517f8462a4b15c2b71eb55655a4ec53281d7fd1e2796778dec5fcc0681094b7  -
```
Because if that's what you did, then you also need the `-L` flag:
```
❯ curl -Ls https://github.com/idris-lang/Idris2/archive/v0.4.0.tar.gz | sha256sum
e06fb4f59838ca9da286ae3aecfeeeacb8e85afeb2e2136b4b751e06325f95fe  -
```
Without `-L` you're just taking the checksum of the redirection stub:
```
❯ curl -s https://github.com/idris-lang/Idris2/archive/v0.4.0.tar.gz
<html><body>You are being <a href="https://codeload.github.com/idris-lang/Idris2/tar.gz/v0.4.0">redirected</a>.</body></html>
```
You can also leave out the `--sha256` flag from `bump-formula-pr` to let `brew` handle this for you.

Reply:
Oops, I definitely did that. Newbie mistake! I got the same (correct) SHA-256 as in your comment above.